### ISSUE TITLE
Unable to vote for commander after reconnect/switch slots/switch teams.

### ISSUE THREAD

**Issue by Gurnhawk** [RSpeekenbrink/ArmA-3-BECTI#11]
**Arma 3 Version:** `x.xx` (stable / rc / dev)

**Mods:**
- `@RHSUSAF`
- `@RHSAFRF`
- `@RHSGREF`

**Description:**
Currently running two missions for Tanoa/Altis with the same problem found here:
http://steamcommunity.com/sharedfiles/filedetails/?id=927447904
http://steamcommunity.com/sharedfiles/filedetails/?id=969373357

The issue is anyone who joins in after the mission has started is unable to vote for commander. They can see the UI for voting but there is only a 0 for their vote. 

**Steps to reproduce:**
Start the mission on a dedicated server. Abort the mission and change teams/player slots or reconnect to the server.

**Where did the issue occur?**
Dedicated server:
{CLAW} Land of Russ.
Contact me for more information if required on steam at:
http://steamcommunity.com/profiles/76561198033896373/

**Comment by snekmate** [comment]
Im having this same problem is there a fix yet?

**Comment by RSpeekenbrink** [comment]
Not yet, I haven't worked on this project for quite a while due to time constrains. Will update this when I find some time.

**Comment by soullesshealer4** [comment]
yeah this is the only major issue that me and my group have ran into sadly. We bought a server and everything as the mod was really high on our list to try out and play for a while. It would mean the world to us if you could fix this soon. Thankyou

**Comment by RSpeekenbrink** [comment]
Currently working on a rewrite since there are a lot of bugs and optimalization issues

**Comment by soullesshealer4** [comment]
thankyou, the other modded versions of becti are dreadful and this was the one we really liked the most. Cannot tell you how much we appreciate this !
